app/views/admins/show_pending_bills.html.erb

```diff
@@ -0,0 +1,13 @@
+Pending Bills | <%= link_to "Back To Admin Page", admins_path() %>
+
+<div>
+  <% @bills.each do |bill|%>
+    <div>
+    <%= bill.date %> <%= bill.price %> <%= bill.facility_id %> <%= bill.procedure_id %> <%= bill.status %>
+    <%= link_to "Approve", approve_bill_admin_path(bill.id), method: :'post' %>
```

Comment:
You shouldn't try to define `post` as both a symbol and a string. You also don't need to specify `bill.id` since Rails can put that together.

```suggestion
    <%= link_to "Approve", approve_bill_admin_path(bill), method: :post %>
```

Reply:
defined post as just a symbol and removed .id